### PR TITLE
feat: added swagger doc for /customers/{cust_id} endpoints

### PR DESCRIPTION
**Pull request by Bobby-Zien**

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -377,26 +377,34 @@ class CustomerResource(Resource):
...
-            raise NotFound(f"Customer with id '{customer_id}' was not found.")
+            raise NotFound("Customer with id '{}' was not found.".format(customer_id))
         return customer.serialize(), status.HTTP_200_OK
 
     #------------------------------------------------------------------
     # UPDATE AN EXISTING CUSTOMER
     #------------------------------------------------------------------
+    @api.doc('update_customers')
+    @api.response(404, 'Customer not found')
+    @api.response(400, 'The posted Customer data was not valid')
```
Comment (ryanjcohen) [inline service/routes.py:399]: I'm wondering if we just want to make sure that the documentation shows similar error messages for different endpoints that can throw the same error code. For example, wondering if we should change Update an existing address' 400 to 'The posted Address data was not valid'

Comment (Bobby-Zien) [reply service/routes.py:399]: Yeah, I think it's better if we can keep consistency. I fine with using posted/supplied. I can change 'The supplied Address data was not valid' to 'The posted Address data was not valid'

Comment (ryanjcohen) [reply service/routes.py:399]: I guess the 404 label is different for customers and addresses too right now, not sure whether we want to worry about consistency there

Comment (Bobby-Zien) [reply service/routes.py:399]: I think it's fine for the address to have a more specific explanations, and i think we don't need to worry about the consistency there.

File: service/routes.py [hunk 2]
```diff
@@ -377,26 +377,34 @@ class CustomerResource(Resource):
...
+    @api.expect(customer_model)
```
Comment (ryanjcohen) [inline service/routes.py:400]: the expected payload should be update_customer_model. everything else looks good otherwise

Comment (Bobby-Zien) [reply service/routes.py]: Agreed. Changed to'update_customer_model'.

**Review by ryanjcohen — APPROVED** [review]

**Comment by ryanjcohen** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/dfa5f370-4610-4cc3-b00a-cc377cfff178?env_id=ibm:yp:us-south "customers-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/bcf906cc-f45b-4b30-a096-6ea68fc0ead1 "customers") deployed [customers](https://github.com/devops-customers-squad/customers/releases/tag/deploy-ibm-yp-us-south-rjc584-nyu.edu-dev-20211204-005505 "deploy-ibm-yp-us-south-rjc584-nyu.edu-dev-20211204-005505") to [dev](https://console.bluemix.net/apps/57207d43-2a44-4c6a-916f-347eb68fb4cc?env_id=ibm:yp:us-south "ibm:yp:us-south:rjc584+@nyu.edu:dev"), including commit 0eed29c98c51d72593ce60ced7ad7ce3c9de44ac

**Pull request merged**